sound/soc/sof/intel/hda.c

```diff
@@ -108,17 +108,20 @@ static void hda_dsp_get_registers(struct snd_sof_dev *sdev,
 				  struct sof_ipc_panic_info *panic_info,
 				  u32 *stack, size_t stack_words)
 {
+	u32 offset = sdev->dsp_oops_offset;
+
 	/* first read registers */
-	sof_block_read(sdev, sdev->mmio_bar, sdev->dsp_oops_offset, xoops,
+	sof_block_read(sdev, sdev->mmio_bar, offset, xoops,
```

Comment:
same here this can't be right, the sizeof below is likely wrong?

Reply:
Yes, same. I started with bdw...